invenio_i18n/__init__.py

```diff
@@ -133,13 +129,47 @@
 or the ``{% trans %}`` tag:
 
 >>> with app.app_context():
-...     r = render_template_string("{% trans %}Long translation{% endtrans %}")
-...     r == "Long translation"
+...     r = render_template_string('{% trans %}Long translation{% endtrans %}')
+...     r == 'Long translation'
 True
 
 For further details and examples see:
 
 * http://jinja.pocoo.org/docs/dev/templates/#i18n
+
+Templates
+---------
+
+Filters
```

Comment:
WDYT if instead of documenting apart the filters we directly enhance the documentation in `jinja2.py`?
Because they are already inserted inside `docs/api.rst`.
In this case we have a single point where the code is documented.